### PR TITLE
[Feature] Init Scripts

### PR DESCRIPTION
**Pull request by pauljugarean**
# Pull Request

ClickUp link: <https://app.clickup.com/t/5amjq7>

## Acceptance criteria

* [ ] Passes all regression tests
* [ ] Is cross-browser compatible
* [ ] Is cross-device compatible
* [ ] Fullfils General Acceptance Criteria (see Wiki)
* [ ] Fullfils Specific Acceptance Criteria (see ClickUp card)
* [ ] Adheres to code standards
* [ ] All code reviews have been addressed
* [ ] No linting errors

QA Wiki: [Generic Criteria](https://wiki.graffino.net/doc/generic-criteria-kHpwt8jdKP)

### CODE REVIEW

File: src/views/layout/header.handlebars [hunk 1]
```diff
@@ -12,23 +12,23 @@
...
+  <meta property="og:description" content="{{package 'description'}}">
   <meta property="og:image" content="/social.jpg">
-  <meta property="og:image:alt" content="Graffino Ninja. Website building webpack flow we use at Graffino.">
+  <meta property="og:image:alt" content="{{package 'description'}}">
 
   <!-- // Twitter Meta -->
   <meta name="twitter:card" content="summary_large_image">
-  <meta name="twitter:title" content="Graffino Ninja">
-  <meta name="twitter:description" content="Graffino Ninja. Website building webpack flow we use at Graffino.">
+  <meta name="twitter:title" content="Graffino proiectu">
```
Comment (zenopopovici) [inline src/views/layout/header.handlebars]:
```suggestion
  <meta name="twitter:title" content="{{package 'name'}}">
```

**Review by zenopopovici — CHANGES_REQUESTED** [review]

File: src/views/layout/header.handlebars [hunk 2]
```diff
@@ -12,23 +12,23 @@
 
   <!-- // Application title - iOS / macOS -->
   <meta name="apple-mobile-web-app-capable" content="yes">
-  <meta name="apple-mobile-web-app-title" content="Graffino Ninja">
+  <meta name="apple-mobile-web-app-title" content="{{package 'description'}}">
```
Comment (Cbooogdan) [inline src/views/layout/header.handlebars]: Cred ca ar trebui sa fie {{package 'name'}}

**Review by Cbooogdan — CHANGES_REQUESTED** [review]

File: bin/clean-ninja.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  await fs.emptyDir(path.resolve(__dirname, '../src/icons'))
+  await fs.emptyDir(path.resolve(__dirname, '../src/fonts'))
+
+  // clean static folder
+  await fs.remove(path.resolve(__dirname, '../src/static/favicon-template.png'))
+  await fs.remove(path.resolve(__dirname, '../src/static/favicon.ico'))
+  await fs.remove(path.resolve(__dirname, '../src/static/pin-icon.svg'))
+  await fs.remove(path.resolve(__dirname, '../src/static/social.jpg'))
+  await fs.remove(path.resolve(__dirname, '../src/static/touch-icon.png'))
+  await fs.remove(path.resolve(__dirname, '../src/static/touch-startup-image.png'))
```
Comment (zenopopovici) [inline bin/clean-ninja.js]: Vezi ca nu-i bine ce faci. Ar trebui sa inlocuiesti astea cu placeholder files (mai ales in static unde oamenii tb sa genereze favicons si socials si trebuie sa aiba un starting point. 

La fel, cred ca in majoritatea folderelor alea trebuie sa fie ceva sa nu dea eroare la build.

File: bin/clean-ninja.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+const path = require('path')
+const fs = require('fs-extra')
+
+// Clean ninja related files
+const cleanFiles = async () => {
+  // clean images, icons and fonts
```
Comment (zenopopovici) [inline bin/clean-ninja.js]: Comentariile incep cu litera mare.

**Comment by zenopopovici** [comment]
Task linked: [TEAM-145 Ninja Init Script](https://app.clickup.com/t/2421429/TEAM-145)

**Comment by zenopopovici** [comment]
Merged.

**Pull request closed without merging**